accumulo/build.sbt

```diff
@@ -7,6 +7,7 @@ libraryDependencies ++= Seq(
     exclude("org.apache.hadoop", "hadoop-client"),
   sparkCore % "provided",
   spire,
+  scaffeine,
```

Comment:
Nope, I expected that as well, but it wasn't the case. I had to add it explicitly to each project.